llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -350,6 +350,11 @@ them.
 
  Extract the named partition from the output.
 
+.. option:: --set-max-section-offset <value>
```

Comment:
Options are listed in alphabetical order. Please fix.

Reply:
Done.